app/controllers/drops_controller.rb

```diff
@@ -4,4 +4,9 @@ def show
     drops = Drop.order("created_at DESC").limit(50).to_json
     render json: drops
   end
+
+  def drop_params
+    params.require(:drop).permit(:photo)
```

Comment:
@r05al we have this in our branch and we have the location permission and the text permission as well